<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pbCoin10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoin50">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoin100">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoin500">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_coffee">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>coffee[100]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tea">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>tea[150]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_milk">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>milk[200]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Reset">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
